lighthouse-core/report/html/report-styles.css

```diff
@@ -951,8 +951,9 @@
 }
 
 .lh-audit-explanation {
-  margin: var(--audit-padding-vertical) 0 calc(var(--audit-padding-vertical) / 2);
+  margin: var(--audit-padding-vertical) 0 calc(var(--audit-padding-vertical) / 2) var(--audit-margin-horizontal);
   line-height: var(--audit-explanation-line-height);
+  display: inline-block;
```

Comment:
what practical effect does this have for our purposes?

Reply:
it's currently a div (w/ display block). this forces a newline

i need it to be inline but also with horizontal margin.

i could change it from a div to a span and _maybe_ drop this `display` prop but w/e